eva_submission/nextflow/accession_and_load.nf

```diff
@@ -103,10 +118,12 @@ workflow {
                 .map{row -> tuple(file(row.vcf_file))}
             accessioned_files_to_rm = Channel.empty()
         } else {
-            valid_vcfs = Channel.fromPath(params.valid_vcfs)
+            normalised_vcfs_ch = Channel.fromPath(params.valid_vcfs)
                 .splitCsv(header:true)
-                .map{row -> tuple(file(row.vcf_file), row.assembly_accession, row.aggregation, file(row.fasta), file(row.report))}
-            accession_vcf(valid_vcfs)
+                .map{row -> tuple(file(row.vcf_file).name, file(row.vcf_file), row.assembly_accession, row.aggregation, file(row.fasta), file(row.report))}
+                .combine(normalise_vcf.out.vcf_tuples, by:0)     // Join based on the vcf file name
+                .map {tuple(it[0], it[7], it[2], it[3], it[4], it[5])}   // filename, normalised vcf, assembly_accession, aggregation, fasta, report
```

Comment:
Maybe call the first element "vcf filename" throughout (including variable names and comments), just to avoid any confusion.

Reply:
I updated the comments and variable names. 
I did tried to use named tuple (or hash table) and it did not work with`combine`